package.json

```diff
@@ -11,6 +11,7 @@
     "react": "^16.4.1",
     "react-apollo": "^2.1.5",
     "react-dom": "^16.4.1",
+    "react-router-dom": "^4.3.1",
```

Comment:
also nother goofy thing about the docker setup, since node_modules is mounted as a volume, when yarn install runs to install new packages, if it's not present locally, it gets blown away when the container is run. :( I need to investigate whats going on with that.